app/core/utils.coffee

```diff
@@ -1,5 +1,75 @@
 slugify = _.str?.slugify ? _.string?.slugify # TODO: why _.string on client and _.str on server?
 
+translatejs2cpp = (jsCode, fullCode=true) ->
```

Comment:
Now that this function works it might be worth writing some unit tests to ensure it works as expected, and continues working into the future.
This should be not too hard to test as it's testing equality of the string returned.

Reply:
good idea!